nvtabular/columns/schema_io/schema_writer_base.py

```diff
@@ -0,0 +1,12 @@
+class SchemaWriter:
```

Comment:
This seems like it might want to be an abstract base class in order to enforce that these methods are overwritten?